Jenkinsfile

```diff
@@ -50,6 +50,11 @@ pipeline {
         }
 
         stage('Release AAB') {
+            // Don't build the release AAB for PRs.
+            when {
+                expression { !params.ghprbPullId }
```

Comment:
I'm new to Jenkins but shouldn't isn't `ghprbPullId` an environment variable? At the bottom of this same Jenkinsfile we have `env.ghprbPullId`.

Reply:
It's actually both. It creates them as parameters dynamically and then also sets them as environment variables since that's typically the only way to access them in a shell script. I forgot I used `env` in the other spot. It's really an unusual plugin. I should probably change it back to `env` for consistency.